src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -319,7 +319,7 @@ internal static dynamic GetPackageJsonObject(ISourceRepo sourceRepo, ILogger log
                 // This prevents Oryx from erroring out when Node.js itself might be able to tolerate the file.
                 logger.LogWarning(
                     exc,
-                    $"Exception caught while trying to deserialize {NodeConstants.PackageJsonFileName}");
+                    $"Exception caught while trying to deserialize {NodeConstants.PackageJsonFileName.Hash()}");
```

Comment:
`package.json` is platform defined file name (nodejs), so should we hash these too?

Reply:
Jennie mentioned that for these platform-defined files that we know the name of, we can create a separate table that maps each file name to a hash that we use when looking at the logs.